server/app/services/agent/node_plugger.rb

```diff
@@ -18,6 +18,7 @@ def plugin!
       begin
         prev_seen_at = node.last_seen_at
         self.update_node
+        self.publish_update_event
```

Comment:
Hmm, why does this need an explicit publish, doesn't the `node.set` trigger the `after_update`?